documentation in sphinx format (#1090)
openvinotoolkit/model_server: docs/clients.md

        data = np.array([1.0, 2.0, ..., 1000.0])
        inputs = {"input_name": data}
        results = client.predict(inputs=inputs, model_name="my_model")

.. tab:: tensorflow-serving-api  

   .. code-block:: python

        import grpc
        from tensorflow_serving.apis import prediction_service_pb2_grpc, predict_pb2
        from tensorflow import make_tensor_proto

        channel = grpc.insecure_channel(„10.20.30.40:9000")
        prediction_service_stub = prediction_service_pb2_grpc.PredictionServiceStub(channel)

        data = np.array([1.0, 2.0, ..., 1000.0])
        predict_request = predict_pb2.PredictRequest()
        predict_request.model_spec.name = "my_model"
        predict_request.inputs["input_name"].CopyFrom(make_tensor_proto(data))
        predict_response = prediction_service_stub.Predict(predict_request, 10.0)
        results = make_ndarray(predict_response.outputs["output_name"])

@endsphinxdirective

For complete usage examples see [ovmsclient samples](../client/python/samples/README.md)

## C++ Client

@sphinxdirective
.. raw:: html

    <div id="switcher-cpp" class="switcher-anchor">C++</div>
@endsphinxdirective

> Creating a client application in C++ follows the same principles as Python, but using C++ adds some complexity. There's no package or library available with convenient functions for interacting with OpenVINO Model Server.

Reviewer: ```suggestion
Creating a client application in C++ follows the same principles as Python, but using C++ adds some complexity. There is no package or library available with convenient functions for interacting with OpenVINO Model Server.
```

To successfully set up communication with the model server you need to implement the logic to communicate with endpoints specified in the [API](api_reference_guide.md). For gRPC, download and compile protos, then link and use them in your application according to the [gRPC API specification](model_server_grpc_api.md). For REST, prepare your data and pack it into the appropriate JSON structure according to the [REST API specification](model_server_rest_api.md).

See our [C++ demo](../example_client/cpp/README.md) to learn how to build a sample C++ client application in a Docker container and get predictions via the gRPC API. 

## Go Client

@sphinxdirective
.. raw:: html

    <div id="switcher-go" class="switcher-anchor">Go</div>
@endsphinxdirective

> Creating a client application in [Go ](https://go.dev/) follows the same principals as Python, but Go adds complexity. There's no Go package or library available with convenient functions for interacting with OpenVINO Model Server..

Reviewer: ```suggestion
Creating a client application in [Go ](https://go.dev/) follows the same principals as Python, but Go adds complexity. There's no Go package or library available with convenient functions for interacting with OpenVINO Model Server.
```

> To successfully set up communication with the model server you need to implement the logic to communicate with endpoints specified in the [API](api_reference_guide.md). For gRPC, download and compile protos, then link and use them in your application according to the [gRPC API specification](model_server_grpc_api.md). For REST, prepare your data and pack it into the appropriate JSON structure according to the [REST API specification](model_server_rest_api.md).

Reviewer: ```suggestion
To successfully set up communication with the model server, you need to implement the logic to communicate with endpoints specified in the [API](api_reference_guide.md). For gRPC, download and compile protos, then link and use them in your application according to the [gRPC API specification](model_server_grpc_api.md). For REST, prepare your data and pack it into the appropriate JSON structure according to the [REST API specification](model_server_rest_api.md).
```
